src/calculator/internal/evaluator.ts

```diff
@@ -113,7 +113,7 @@ export default function evaluate(tokens: Token[], ans: Decimal, ind: Decimal, an
 							return ok(
 								radicand.isZero()
 									? radicand
-									: radicand.lt(0) && !degree.mod(2).eq(0)
+									: radicand.lt(0) && degree.isInteger() && !degree.mod(2).eq(0)
```

Comment:
A negative number has a real nth root iff n is odd; this ternary just tries to evaluate a negative radicand as if it were positive whenever the degree isn't odd. Probably better to have a separate if-statement for this; I can whip it up quickly.

Reply:
```typescript
if (degree.isZero()) {
  return err("NOT_A_NUMBER" as const);
}

if (radicand.isZero()) {
  return ok(ZERO);
}

if (radicand.isPositive()) {
  return ok(radicand.pow(ONE.div(degree)));
}

if (radicand.isNegative() && degree.isInteger() && degree.mod(2).eq(1)) {
  return ok(radicand.neg().pow(ONE.div(degree)).neg());
}

return err("NOT_A_NUMBER" as const);
```